07week/towersOfHanoi/script.js

```diff
@@ -3,21 +3,62 @@
 class TowersOfHanoi extends React.Component {
   constructor(props) {
     super(props);
+    this.state = { 
+      block: null, 
+      board: {
+        1: [100,75, 50,25],
+        2: [],
+        3: []
+      }
+    };
+    this.playerTurn = this.playerTurn.bind(this);
+  }
+  // move disks on board
+  playerTurn(event){
+      const board = this.state.board; 
+      const block = this.state.block;
+      const stack = event.target.getAttribute("data-stack"); 
+      const selStack = board[stack] // selected stack ie this.state.board[1]
+      const store = []; // 
+
+      // add disk to stack - onClick
+      if (!block) { // if the stack has a block
+        const move = selStack.pop(); // take last element in selected board's array
+        this.state.block = move; // move to this.state.block
+        store.push(move); // move to store
+        // console.log(this.state); // console object
+        // console.log("if runs"); // console test
+      } else if (selStack.length === 0 || selStack[selStack.length-1] > block) { // if nothing in stack, or last item in stack is greater than stored block
```

Comment:
You don't allow me to make an invalid move, but a message telling me the move was invalid would have been nice.

Nice job